Improbable/Postgres/Improbable.Postgres.CSharpCodeGen/ComponentGenerator.cs

```diff
@@ -121,32 +117,30 @@ private static string PostgresTypeConversion(FieldDefinition f)
             switch (f.TypeSelector)
             {
                 case FieldType.Option:
-                    switch (f.OptionType.InnerType.ValueTypeSelector)
+                    if (f.OptionType.InnerType.ValueTypeSelector == ValueType.Primitive)
```

Comment:
I think you can use if switch `when` here to remove the outmost if

Reply:
Ah yes, nice!